boards/xtensa/intel_adsp_cavs15/tools/adsplog.py

```diff
@@ -40,31 +40,39 @@
 WIN_SIZE = 0x20000
 LOG_OFFSET = WIN_OFFSET + WIN_IDX * WIN_SIZE
 
-# List of known ADSP devices by their PCI IDs
-DEVICES = ["8086:5a98"]
-
 mem = None
+pcid="/sys/bus/pci/devices"
+
+for fname in os.listdir(pcid):
+    pfile = pcid + "/" + fname + "/class"
```

Comment:
Sorry to be demanding but could we have a little bit more... "imagination" for the variable names to make this a bit faster to read? How about `dev_addr` instead of `fname`, `sys_devices` instead of `pcid`, `class_file` instead of `pfile`,...?

Reply:
Ack, will be renamed in next version.